specification/web/resource-manager/Microsoft.Web/stable/2021-03-01/KubeEnvironments.json

```diff
@@ -328,6 +328,68 @@
         }
       }
     },
+    "ContainerAppsConfiguration": {
+      "type": "object",
+      "properties": {
+        "daprAIInstrumentationKey": {
+          "type": "string",
+          "description": "Azure Monitor instrumentation key used by Dapr to export Service to Service communication telemetry",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "vnetConfiguration": {
```

Comment:
In the current CLI custom sdk, there isn't `vnetConfiguration` property, but there are `appSubnetResourceId` and `controlPlaneSubnetResourceId` properties